subprojects/docs/src/docs/userguide/publishingMaven.adoc

```diff
@@ -213,6 +213,34 @@ The created task is named “`publish«_PUBNAME_»PublicationToMavenLocal`”.
 
 The resulting task in this example is named “`publishMavenJavaPublicationToMavenLocal`”. This task is wired into the `publishToMavenLocal` lifecycle task. Executing “`gradle publishToMavenLocal`” builds the POM file and all of the artifacts to be published, and “installs” them into the local Maven repository.
 
+[[publishing_maven:conditional_publishing]]
+=== Conditional publishing
+
+When you have defined multiple publications or repositories, you often want to control which publications are published to which repositories. For instance, consider the following sample that defines two publications and two repositories:
+
+++++
+<sample dir="maven-publish/conditional-publishing" id="multiplePublicationsAndRepositories" title="Adding multiple publications and repositories">
+    <sourcefile file="build.gradle" snippet="publishing"/>
+</sample>
+++++
+
+You can configure the tasks generated by the “`maven-publish`” plugin to be skipped based on certain criteria. The following sample demonstrates how to publish the `binary` and `binaryAndSources` publications only to the `external` and `internal` repositories, respectively. In addition, it configures only `binaryAndSources` to be published to the local Maven repository.
```

Comment:
I'm currently using title case for plugin names, with "Plugin" being part of the name. Not a terribly popular approach, but it's the consistency that counts :smile: Anyway, please use Maven Publish Plugin rather than “\`maven-publish`”. And please update other such references in the chapter if you can. I think I remember this chapter using `maven-publish` originally.

Reply:
I would prefix "You can configure..." paragraph with another that explains the reasoning for doing so, i.e. that the plugin automatically generates tasks for all the combinations. For example, you could try:

    You may not want build users publishing both types of publication to both repositories, but the plugin automatically generates tasks for all possible combinations. So how do you stop someone from publishing the `binaryAndSources` publication to the `external` repository?

I also think it's easier to understand if you say:

    ...to restrict the `binary` publication to the `external` repository and the `binaryAndSources` publication to the `internal` repository.

It's more typing, but there's less scope for confusion.

Reply:
As for replacing ``“`maven-publish`”``: I'm all for it! However, I'd rather submit separate PR for both Maven and Ivy Publishing Plugins.

As for the rest: I'll add/change it. 👍